### ISSUE TITLE
3D map error

### ISSUE THREAD

**Issue by michelleasaurus** [uncovery/uncovery_me#308]
Luke and I were in Kingdom (at y:15), I checked the 3D map and noticed we were't on it. 
Other players were still on the map and moving around. 

I left to Empire and Luke followed. 
He then registered on the map, but I still didn't.

**Comment by uncovery** [comment]
I can check if there is a new version of the 3D map, but if not then I won't be able to fix that.

**Comment by michelleasaurus** [comment]
I had assumed it was beyond our reach. Just wanted to inform. I have been checking back every so often. I still don't register on the maps. 

* this is not a issue for me personally, but it might be bad for new players.

**Comment by uncovery** [comment]
Yeah I cannot fix that. I have no idea what's wrong there and debugging the dynmap is out of my capabilities. I will leave the bug open here for others to see, but I label it "won't fix .
